src/main/kotlin/no/nav/familie/ba/sak/beregning/TilkjentYtelseService.kt

```diff
@@ -44,7 +44,7 @@ class TilkjentYtelseService(private val satsService: SatsService) {
                     innvilgetPeriodeResultatSøker
                             .filter { it.overlapper(periodeResultatBarn) }
                             .flatMap { overlappendePerioderesultatSøker ->
-                                val person = identBarnMap[periodeResultatBarn.personIdent]!!
+                                val person = identBarnMap[periodeResultatBarn.personIdent]!! // TODO: bør vi kaste error her eller?
```

Comment:
Hva tenker du, at vi har `throw IllegalStateException`-ish her?

Sett fra "systemperspektiv" for min del så skal vel alle barna som har perioderesultater eksistere i personopplysningsgrunnlaget. Ergo BØR ikke denne feile, men teknisk sett kan den jo det.

Reply:
```suggestion
                                val person = identBarnMap[periodeResultatBarn.personIdent] ?: error("Feilmelding")
```

Ja, kunne gjort noe slikt f.eks.? Enig i at det ikke burde skje, jeg har bare litt ocd mot feilmeldinger i IDE'n ;)

Reply:
Ja, kanskje du kan legge inn den endringen direkte i denne PR-en? Jeg støtter forslaget - enig i at eksplisitte feil trumfer implisitte feil.

Reply:
Yes, jeg kan legge det til 👍